### DISCUSSION TITLE
widget support

### DISCUSSION THREAD

**Discussion by DoStraTech** [drengskapur/colab2pdf#2] [Ideas]
The code does not seem to work with widgets. It only prints the code.
This would be a nice feature, since it seems hard to print the widgets any other way.

**Comment by jonathanagustin** [comment]
@DoStraTech - I'll see what I can do! I haven't made a widget before - I'll research it and try implementing it.

**Reply by jonathanagustin** [reply]
#3

**Comment by jonathanagustin** [comment]
@DoStraTech Implemented as widget - great idea btw. It makes things easier.
